unified-scheduler-pool/src/lib.rs

```diff
@@ -2672,7 +2673,9 @@ mod tests {
             .take(10000)
         {
             let scheduler = pool.take_scheduler(context.clone());
-            scheduler.schedule_execution(&(dummy_tx, index)).unwrap();
+            scheduler
+                .schedule_execution(dummy_tx.clone(), index)
```

Comment:
nit: seems this `.clone()` isn't needed anymore.

Reply:
i miss clippy's `redundant_clone`... ;)

Reply:
this one is necessary because it's in a loop